src/Compilers/Core/AnalyzerDriver/AnalyzerManager.cs

```diff
@@ -265,12 +265,6 @@ internal bool IsSupportedDiagnostic(DiagnosticAnalyzer analyzer, Diagnostic diag
 
             foreach (var diag in supportedDiagnostics)
             {
-                if (HasNotConfigurableTag(diag.CustomTags))
```

Comment:
what happen if not-configurable, off by default, default severity is error? @mavasani